NEWS.md

```diff
@@ -0,0 +1,53 @@
+# Changes by Version
+
+## [Unreleased]
+
+### Added
+
+* Add `tmpdir` parameter to specify a custom temp directory (#230)
```

Comment:
> Just have to be careful not to prematurely click the big green button on the page (why do they have to always have tempting big green buttons?!)

Yeah, today I already accidentally clicked the "close PR" button.

The nice thing about the `NEWS.md` approach is that a user can tell what's unreleased - using just GitHub Releases doesn't really help with that use case. I have needed to figure out if feature X has been released or not when responding to issues.